docs/documentation/updating-the-kit.md

```diff
@@ -13,43 +13,41 @@ Tell us as much as you can about the issue you're having, and the computer and o
 
 ## Updating to the latest version
 
-1. [Download the zip file of the latest version of the Prototype Kit](/docs/download).
+### Find what version you’re using
 
-2. Unzip the zip file. It will make a folder called `govuk-prototype-kit`, with a version number.
+In Finder on Mac or Windows Explorer go to your prototype folder and open the file `VERSION.txt`. This will show what version of the prototype kit you’re using.
 
-3. Make a backup copy of your prototype folder.
+- If your prototype is version 8 or before, then contact the GOV.UK Prototype team for help with updating it
+- If your prototype is version 9 or 10, you can update to version 11 by following the steps on this page
 
-4. You’ll need to copy some files that operating systems do not usually show. These hidden files usually start with a `.` and are known as ‘dot files’. You can:
-    - [show hidden files in Windows](https://support.microsoft.com/en-us/windows/view-hidden-files-and-folders-in-windows-97fbc472-c603-9d90-91d0-1166d1d9f4b5)
-    - show hidden files on MacOS by pressing command + shift + . in Finder 
+### Updating to version 11
 
-5. In your prototype folder, delete everything except the `app` and `.git` folders.
+1. Make a backup copy of your prototype folder.
 
-6. Copy all the files and folders except the `app` folder from the `govuk-prototype-kit` folder to your prototype.
+2. Make a backup of your prototype folder. You can do this in Finder or Windows Explorer. This may take a few minutes. In the terminal, `cd` to your prototype folder.
 
-7. Replace the `app/config.js` file in your prototype with the `app/config.js` file from the `govuk-prototype-kit` folder.
+3. Run this command:
+ 
+```
+curl -L https://govuk-prototype-kit.herokuapp.com/docs/update.sh | bash
+```
 
-8. Compare your new `config.js` file to the `config.js` file in the backup you made in step 3, and copy over anything you need to from the backup - for example your service name.
 
-9. Copy `app/assets/sass/patterns` from the `govuk-prototype-kit` folder to your prototype.
+4. In a code editor (like Atom) open the file at `update/app/assets/sass/application.scss`.
 
-10. Open the `app/assets/sass/application.scss` file in the `govuk-prototype-kit` folder.
+5. Copy everything until the line that starts with `// Add extra styles here`. 
 
-11. Copy everything down to `// Add extra styles here`, then paste it into the `app/assets/sass/application.scss` file in your prototype so it replaces everything above `// Add extra styles here`.
+6. Open the file at `app/assets/sass/application.scss`. This is the file in your prototype folder, not the one in the update folder. 
```

Comment:
This might be a personal preference, but I found it mildly confusing that even though we give the file path to the folder in step 4, we then here start referring to it as the 'update folder'. Would it be clearer to change this to
> This is the file in `app/`, not the one in `update/`

Reply:
I think this is the version that tested best with users, so reluctant to change it.

Reply:
No worries. 

An optional idea might be to add something to step 3. to say that the command will create a folder called 'update' inside the prototype. It might make the process and later references clearer. Plus even I had to rewind for a second  when looking inside the prototype and thinking to myself 
> Did we always have a folder called 'update' in the kit?

before realising that the script had created it.

Reply:
Ah, interesting, I think there was a sentence about this in step 3 in the draft, but it appears to have gotten missed, I will add it in.